CHANGELOG.md

```diff
@@ -92,6 +92,7 @@ You can find its changes [documented below](#070---2021-01-01).
 
 ### Fixed
 
+- Centre checkmark in checkbox ([#2036] by [@agentsim])
```

Comment:
You'll also need to define the links to 2036 and agentsim at the bottom of the file.

Reply:
Thanks, fixed.